run.sh

```diff
@@ -30,6 +30,7 @@ PODMAN_ARGS=(
     --userns=keep-id
     --mount type=bind,source=${HOME},target=${HOME}
     --mount type=bind,source=$(pwd),target=/waterneuron
+    --mount type=bind,source=$(pwd),target=/boomerang
```

Comment:
you don't need to mount on `/boomerang`, you're mounting your current directory twice